app/views/govuk_publishing_components/components/_government_navigation.html.erb

```diff
@@ -25,11 +25,6 @@
       </a>
     </li>
     <li class="clear-child">
-      <a class="<%= 'active' if active == 'publications' %>" href="/government/publications">
-        <%= t("govuk_component.government_navigation.publications", default: "Publications") %>
```

Comment:
Is there a translation we can remove as well?

Reply:
This repo doesn't seem to have a translation for this.